x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/ShardChangesActionTests.java

```diff
@@ -147,7 +147,7 @@ public void testGetOperationsExceedByteLimit() throws Exception {
         final IndexShard indexShard = indexService.getShard(0);
         final Translog.Operation[] operations = ShardChangesAction.getOperations(indexShard, indexShard.getLastKnownGlobalCheckpoint(),
             0, 12, indexShard.getHistoryUUID(), new ByteSizeValue(256, ByteSizeUnit.BYTES));
-        assertThat(operations.length, equalTo(12));
+        assertThat(operations.length, equalTo(11));
```

Comment:
For my context, why did this number change?

Reply:
The test is checking what happens when documents stored in the translog take up more than a certain amount of space, in this case 256 bytes.  Previously, the documents had a type of `doc`, but now they have a type of `_doc` (as a default - a future PR will remove type entirely from the doc stored in the translog), which is one byte longer, so we fit one fewer document in the same number of bytes.

Reply:
Thanks for the explanation.